shared-libs/outbound/package-lock.json

```diff
@@ -5,7 +5,386 @@
   "requires": true,
   "dependencies": {
     "@medic/settings": {
-      "version": "file:../settings"
+      "version": "file:../settings",
```

Comment:
I think you need to leave this dependency as it it, it might give you error when installing with `npm ci`